pkg/remote/aws/lambda_event_source_mapping_supplier.go

```diff
@@ -0,0 +1,69 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+
```

Comment:
There is spaces between your imports (not only in this file)

Reply:
yeah it's go import way or organizing import ?

Reply:
Hmm maybe it's goland I don't really know I removed them

Reply:
I don't know what is the standard way to organize imports, but we should follow it and especially we should all use the same logic.

By default it seems that Goland split stdlib pkg (like `fmt`) from others and is using gofmt.

![image](https://user-images.githubusercontent.com/6154987/109651922-6c0f8c80-7b5f-11eb-8237-69bf83905b2d.png)

When using `go fmt -local` it groups imports by pkg and add whitespaces between them.

I also have goimport as file watcher in goland and the same gofmt logic seems to be applied.

I don't really care which option we take but we should avoid unnecessary conflicts in imports.

Reply:
yeah I didn't see that go fmt had an other way of organizing, I removed the newline but I have no preference on what to use